doc/man7/EVP_PKEY-EC.pod

```diff
@@ -110,7 +110,9 @@ per-operation basis.
 
 =item "pub" (B<OSSL_PKEY_PARAM_PUB_KEY>) <octet string>
 
-The public key value in EC point format.
+The public key value in encoded EC point format. This parameter is used
+when importing or exporting the public key value with the EVP_PKEY_fromdata()
+and EVP_PKEY_todata() functions.
 
```

Comment:
The description of `OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY` below might need a tweak to say it is importable (at the moment it just says it is used for "getting and setting").

Reply:
I'm not sure. I see the OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY only as a fallback importable parameter. I'd even support dropping the last commit from this PR altogether.

Reply:
Yeah - dropping it may be the right answer. It's a shame that we don't have separate importable/exportable lists. But that last commit makes `OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY` into a special category all of its own which might not be a good idea (especially considering that this PR is targetting 3.0 as well)

Reply:
I've dropped the commit with the fallback. Please re-review.